defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+__elasticsearch_version: "2.x"
```

Comment:
You can keep these two variables as normal variables without the preceding `__`—they're new, and they can easily be overridden since they're in defaults.